packages/components/src/components/RadioGroup/components/Radio/Radio.module.scss

```diff
@@ -61,13 +58,13 @@
       "label";
     grid-template-columns: 1fr;
     justify-items: center;
-    border-radius: var(--radio-group--corner-radius);
+    border-radius: var(--form-control--corner-radius);
     .label {
-      font-weight: var(--radio-group--label-font-weight-with-content-or-icon);
+      font-weight: var(--font-weight--bold);
     }
     .icon {
-      height: var(--radio-group--icon-size-without-content);
-      width: var(--radio-group--icon-size-without-content);
+      height: var(--icon--size--l);
+      width: var(--icon--size--l);
```

Comment:
Du nimmst hier jetzt globale Design-Tokens. Würde aber sagen, dass wir hier die Design-Entscheidung für den Radio-Button weiterhin in den Component-Design-Token festhalten.